osmaxx/excerptexport/models/export.py

```diff
@@ -47,16 +51,23 @@ def get_full_status_update_uri(self, request):
     def status_update_url(self):
         return reverse('job_progress:tracker', kwargs=dict(export_id=self.id))
 
-    def set_and_handle_new_status(self, new_status):
+    def set_and_handle_new_status(self, new_status, *, incoming_request):
         if self.status != new_status:
```

Comment:
Fixed in a sub-PR (i.e., one that'd merge into `feature/#438-handle_status_updates`): #462